creator-node/src/components/replicaSet/exportComponentService.js

```diff
@@ -99,7 +99,6 @@ const exportComponentService = async ({
       const curCnodeUserClockVal = cnodeUser.clock
 
       // Resets cnodeUser clock value to requestedClockRangeMax to ensure consistency with clockRecords data
-      // Also ensures secondary knows there is more data to sync
       if (cnodeUser.clock > requestedClockRangeMax) {
         // since clockRecords are returned by clock ASC, clock val at last index is largest clock val
         cnodeUser.clock = requestedClockRangeMax
```

Comment:
basically not to mutate cnodeUser.clock and store it in a variable like `exportMaxClockVal` or something.

Reply:
so basically only set it in the return object?

Reply:
oh yeah i was getting confused

Reply:
fixed